src/rtc.c

```diff
@@ -355,7 +355,7 @@ void RTC_init(hourFormat_t format, sourceClock_t source, bool reset)
   RTC_DateTypeDef BackupDate;
```

Comment:
The RTC_DateTypeDef is made of date (day) month, year which are (8bit values) usually stored in 3 backup 16bit registers.
Some explanations could help to see how that date is combined saved/retrieve on the different BKUP registers